src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -196,6 +196,13 @@
     <Field type="GCWhen" name="when" label="When" />
     <Field type="VirtualSpace" struct="true" name="heapSpace" label="Heap Space" />
     <Field type="ulong" contentType="bytes" name="heapUsed" label="Heap Used" description="Bytes allocated by objects in the heap" />
+    <Field type="ulong" contentType="bytes" name="heapLive" label="Heap Live" description="Live bytes allocated by objects in the heap" />
+  </Event>
+
+  <Event name="HeapUsageSummary" category="Java Virtual Machine, GC, Heap" label="Heap Usage Summary" startTime="false" period="everyChunk">
+    <Field type="ulong" contentType="bytes" name="capacity" label="Heap Capacity" description="Maximum number of bytes to be allocated by objects in the heap" />
+    <Field type="ulong" contentType="bytes" name="used" label="Heap Used" description="Bytes allocated by objects in the heap" />
+    <Field type="ulong" contentType="bytes" name="live" label="Heap Live" description="Live bytes allocated by objects in the heap" />
```

Comment:
I think it would be good to mention in the description that it is an estimate, i.e. "Estimate of live bytes ....".

Reply:
:+1: